### ISSUE TITLE
public local occupancy map by only using stereo camera

### ISSUE THREAD

**Issue by Aber111** [introlab/rtabmap#1185]
Hello! 
Thank you very much for your work!
Can rtabmap generate a local 2D occupancy map with only stereo cameras?
Looking forword to your reply, Thanks!

**Comment by matlabbe** [comment]
Yes, it does. If you subscribe only to stereo topics, by default it will create one. If you subscribe to a lidar at the same time, you have to explicitly set `Grid/Sensor=1` to use the camera to create the occupancy grid. If you are navigating in 2D, you could set `Grid/3D=false` and `Grid/Raytracing=true` to fill more unknown space.
